lib/ssh/src/ssh_sftpd.erl

```diff
@@ -676,12 +677,6 @@ do_open(ReqId, State0, Path, Flags) ->
 	    ssh_xfer:xf_send_status(State0#state.xf, ReqId,
```

Comment:
This case clause must also return `State0` to avoid bad return from  `handle_op/4`.